TVToolBoxCInterface.hs

```diff
@@ -0,0 +1,19 @@
+module TVToolBoxCIntegererface where
+
+import TVToolBox
+
+import Foreign.C.Types
+
+--Compute Jensen-Shannon divergence given bin,TV,Count and lookahead values.
+sqrtJsdC :: Integer -> MyFloat -> Integer -> Integer -> Integer -> MyFloat -> Integer -> Integer -> MyFloat
+sqrtJsdC b1 s1 c1 k1 b2 s2 c2 k2 = sqrtJsd d1 d2
+	where d1 = discretize b1 (genDist s1 c1 k1)
+              d2 = discretize b2 (genDist s2 c2 k2)
```

Comment:
k should be the same for the 2 distributions, otherwise the comparison might not be faithful. For instance if the 2 distributions originate from the same strength and count but difference lookaheads, their sqrtJsd will be greater than zero, which isn't what we want.